misc/output_test.py

```diff
@@ -130,6 +130,31 @@ def test_pr_1685(self):
         self.assertEqual(run('', flags='-t recompact'), '')
         self.assertEqual(run('', flags='-t restat'), '')
 
+    def test_pr_2390(self):
```

Comment:
Can you renamed this `test_issue_2048` instead? Using an issue number makes more sense than a pull request one in this context.